dotnetv3/Polly/SynthesizeSpeechExample/SynthesizeSpeech.cs

```diff
@@ -39,18 +37,18 @@ static async Task Main()
         /// <param name="text">The text which will be converted to speech.</param>
```

Comment:
<param name="text">The text to convert to speech.</param>

Reply:
done